### PR TITLE
Room queue in ui

### LINKED ISSUE #29: Display the room queue on the /rooms/<code> page

### LINKED ISSUE #34: Change response code from successful api/rooms/<code>/queue
from 202 to 200 is probably correct

### PR DESCRIPTION
**Pull request by DanLindeman**
Closes #29 
Closes #34

### CODE REVIEW

File: kcards/views/room.py [hunk 1]
```diff
@@ -16,3 +16,8 @@ def detail(code):
     response = Response(render_template("room.html", room=content))
 
     return response
+
+
+@blueprint.route("/<code>/next", methods=['POST'])
```
Comment (jacebrowning) [inline kcards/views/room.py]: Isn't this going to be an API view?

File: kcards/views/api_rooms.py [hunk 2]
```diff
@@ -74,4 +74,4 @@ def queue(code):
     content = room.data
     content['uri'] = url_for('api_rooms.detail', code=room.code, _external=True)
 
-    return content, status.HTTP_202_ACCEPTED
+    return content, status.HTTP_200_OK
```
Comment (jacebrowning) [inline kcards/views/api_rooms.py]: 👍

**Comment by jacebrowning** [comment]
I updated your PR description to reference the issues this is related to.

https://help.github.com/articles/closing-issues-via-commit-messages

File: kcards/views/room.py (hunk 1 above)
Comment (DanLindeman) [inline kcards/views/room.py]: I haven't figured out how to redirect this to the api_rooms yet, but agreed

**Pull request merged**